Sources/Profile-Modify.php

```diff
@@ -2897,7 +2899,7 @@ function loadThemeOptions($memID)
 			WHERE id_theme IN (1, {int:member_theme})
 				AND id_member IN (-1, {int:selected_member})',
 			array(
-				'member_theme' => (int) $cur_profile['id_theme'],
+				'member_theme' => !isset($cur_profile['id_theme']) && $defaultSettings ? 0 : (int) $cur_profile['id_theme'],
```

Comment:
Please change this to `!isset($cur_profile['id_theme']) && !empty($defaultSettings)`, just in case someone passes a goofy value to the `$defaultSettings` parameter.